Makefile

```diff
@@ -634,6 +634,7 @@ helm-repo-add:
 # create istio-remote.yaml
 istio-remote.yaml: $(HELM) $(HOME)/.helm helm-repo-add
 	cat install/kubernetes/namespace.yaml > install/kubernetes/$@
+	cat install/kubernetes/helm/istio-init/files/crd-* >> install/kubernetes/$@
```

Comment:
all of these sections are redundant and can be merged into one rule.  Will do that as a followup PR.